### DISCUSSION TITLE
What am i doing wrong? May i ask for help?

### DISCUSSION THREAD

**Discussion by castleclashnuut** [UltimaHoarder/UltimaScraper#1930] [Q&A]
i have installed everything and edited the auth file with the details from the onlyfans page after i logged in. it keeps erroring on me
![1](https://user-images.githubusercontent.com/114783304/193368275-c21b67f7-026f-45b7-93d8-29eff6cd0c44.png)
![2](https://user-images.githubusercontent.com/114783304/193368277-9eb139e7-af35-45f7-81a1-c00fc3c7d312.png)
![3](https://user-images.githubusercontent.com/114783304/193368278-ac4bcf9b-2a57-41d2-b408-4b7862d4a503.png)
![4](https://user-images.githubusercontent.com/114783304/193368279-c9b925ec-5a00-4975-a148-50210128ea8d.png)

**Comment by dnsamw** [comment]
press 0 and hit enter
